src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -3483,7 +3484,6 @@ public void testNetworks() throws Exception {
     final IpamConfig ipamConfig =
         IpamConfig.create("192.168.0.0/24", "192.168.0.0/24", "192.168.0.1");
     final Ipam ipam = Ipam.builder()
-        .driver("default")
```

Comment:
I think you’re right. I was thinking that, since I could remove the ipam config in commit [dcec3e4](https://github.com/spotify/docker-client/pull/676/commits/dcec3e4eb052b20ef735cfb0cf1e554c475ac803), that I could also remove this here. But I think that those are two different cases: the former works, but this one does not.